modules/articles/client/config/articles.client.routes.js

```diff
@@ -12,25 +16,56 @@ angular.module('articles').config(['$stateProvider',
       })
       .state('articles.list', {
         url: '',
-        templateUrl: 'modules/articles/client/views/list-articles.client.view.html'
+        templateUrl: 'modules/articles/client/views/list-articles.client.view.html',
+        controller: 'ArticlesListController',
+        controllerAs: 'vm'
       })
       .state('articles.create', {
         url: '/create',
-        templateUrl: 'modules/articles/client/views/create-article.client.view.html',
+        templateUrl: 'modules/articles/client/views/form-article.client.view.html',
+        controller: 'ArticlesController',
+        controllerAs: 'vm',
+        resolve: {
+          articleResolve: newArticle
```

Comment:
I am not sure I understand the concern?  We are resolving a new instance of an article which is being injected to the controller.  Unless you want to have a separate controller for create vs edit/view, you need to resolve something.  One can argue that you can resolve null and then check for null in the controller and instantiate there, but what is the point when you can avoid it?  Adding another controller will just duplicate code.